### ISSUE TITLE
Static web-indexable archives

### ISSUE THREAD

**Issue by erlend-sh** [commune-os/commune-server#18]
Let’s say a Commune space stops being active. They want to keep their history around for posterity, but they don’t want to incur the costs of running a Matrix server any longer. There should be a way for them to keep a static export of their space.

We have a preliminary version of this working:
https://archive.commune.sh/

Ideally it wouldn’t even require a different frontend. It’d look roughly the same as a live Commune instance, just functionally degraded of course; practically no dynamic functions any longer. Perhaps a client-side search if the archive isn’t too big.

Basically every space should have it's archive endpoint that's read only. So archive.commune.sh/rust or similar.
